modular_skyrat/modules/specialist_armor/code/cargo_packs.dm

```diff
@@ -0,0 +1,32 @@
+/datum/supply_pack/security/armory/soft_armor
+	name = "Soft Armor Kit Crate"
+	crate_name = "soft armor kit crate"
+	desc = "Contains three sets of SolFed-made soft body armor and matching helmets."
+	access = ACCESS_MEDICAL
+	cost = CARGO_CRATE_VALUE * 5
+	contains = list(
+		/obj/item/clothing/head/helmet/sf_peacekeeper/debranded = 3,
+		/obj/item/clothing/suit/armor/sf_peacekeeper/debranded = 3,
+	)
+
+/datum/supply_pack/security/armory/hardened_armor
+	name = "Hardened Armor Kit Crate"
+	crate_name = "hardened armor kit crate"
+	desc = "Contains three sets of SolFed-made hardened body armor and matching helmets."
+	access = ACCESS_MEDICAL
+	cost = CARGO_CRATE_VALUE * 5
+	contains = list(
+		/obj/item/clothing/head/helmet/toggleable/sf_hardened = 3,
+		/obj/item/clothing/suit/armor/sf_hardened = 3,
+	)
+
+/datum/supply_pack/security/armory/sacrificial_armor
+	name = "Sacrificial Armor Kit Crate"
+	crate_name = "sacrificial armor kit crate"
+	desc = "Contains three sets of SolFed-made sacrificial body armor and matching helmets."
+	access = ACCESS_MEDICAL
+	cost = CARGO_CRATE_VALUE * 5
```

Comment:
Same comments here.

Reply:
And considering that the sacrificial armor physically breaks and falls off you without any ability to repair it, someone who's not into that kinda thing would choose one of the other armors.